libs/designsystem/src/lib/components/progress-circle/progress-circle.component.ts

```diff
@@ -0,0 +1,77 @@
+import {
+  Component,
+  Input,
+  ChangeDetectionStrategy,
+  HostBinding,
+  ElementRef,
+  AfterViewInit,
+  ChangeDetectorRef,
+  OnDestroy,
+} from '@angular/core';
+
+const RADIUS_MAP = {
+  sm: 20,
+  md: 28,
+  lg: 48,
+};
+
+@Component({
+  selector: 'kirby-progress-circle',
+  templateUrl: './progress-circle.component.html',
+  styleUrls: ['./progress-circle.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class ProgressCircleComponent implements AfterViewInit, OnDestroy {
+  @Input() value: number = 0;
+  @Input() size: 'sm' | 'md' | 'lg' = 'md';
+  @Input() themeColor: 'success' | 'warning' | 'danger';
```

Comment:
Consider applying default here instead of in scss:
```suggestion
  @Input() themeColor: 'success' | 'warning' | 'danger' = 'success';
```

Reply:
Fixed